NumLib/TimeStepping/Algorithms/IterationNumberBasedTimeStepping.cpp

```diff
@@ -119,6 +126,35 @@ double IterationNumberBasedTimeStepping::getNextTimeStepSize() const
     return dt;
 }
 
+double IterationNumberBasedTimeStepping::checkSpecificTimeReached(const double h_new)
```

Comment:
The method changes the `_fixed_output_times` vector but the name doesn't is only checkSomething(). For me it is not obvious what the method does and the there isn't any documentation.

Reply:
My copy-paste laziness ;)
Fixing...